instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/otel/OtelJvmHeapPressureMetrics.java

```diff
@@ -72,28 +57,15 @@ private OtelJvmHeapPressureMetrics(Duration lookback, Duration testEvery) {
     this.gcPauseSum =
         new TimeWindowSum((int) lookback.dividedBy(testEvery.toMillis()).toMillis(), testEvery);
 
-    longLivedPoolNames =
-        JvmMemory.getLongLivedHeapPools()
-            .map(MemoryPoolMXBean::getName)
-            .collect(Collectors.toSet());
-
     monitor();
   }
 
   public void install() {
     Meter meter = OtelMeterProvider.get();
 
-    if (!longLivedPoolNames.isEmpty()) {
-      Attributes attributes = Attributes.of(AREA, "heap", POOL, "long-lived");
-
-      meter
-          .gaugeBuilder("runtime.jvm.memory.usage.after.gc")
-          .setUnit("percent")
-          .setDescription(
-              "The percentage of long-lived heap pool used after the last GC event, in the range [0..1].")
-          .buildWithCallback(
-              measurement -> measurement.record(lastLongLivedPoolUsageAfterGc.get(), attributes));
-    }
+    // runtime.jvm.memory.usage.after.gc is replaced by OTel
+    //   process.runtime.jvm.memory.usage_after_last_gc{pool=<long lived pools>,type=heap} /
+    //   process.runtime.jvm.memory.limit{pool=<long lived pools>,type=heap}
 
     meter
         .gaugeBuilder("runtime.jvm.gc.overhead")
```

Comment:
The `runtime.jvm.gc.overhead` metrics does not have any matching counterpart on the OTel side, but I think that we should probably remove that: this sort of aggregation (% of time used for GC in the last 5 minutes) should be done in dashboards, not in the instrumentation code.

Reply:
removed then ✂️